playbooks/group_vars/all.yml

```diff
@@ -3,18 +3,28 @@
 ################################################################################
 
 ################################################################################
-# - The value for property BPXK_AUTOCVT must be configured to ON.
+# - The value for property BPXK_AUTOCVT must be configured to 'ON'.
 # - The value for property ZOAU_HOME is the ZOA Utilities install root path,
-# for example: /usr/lpp/IBM/zoautil.
+#   for example: /usr/lpp/IBM/zoautil.
 # - The value for property ZOAU_ROOT is the ZOA Utilities install root path,
-# for example: /usr/lpp/IBM/zoautil.
+#   for example: /usr/lpp/IBM/zoautil.
 # - The value for property PYTHONPATH is the ZOA Utilities Python library path,
-# for example: /usr/lpp/IBM/zoautil/lib/.
+#   for example: /usr/lpp/IBM/zoautil/lib/.
 # - The value for property LIBPATH is both the path to the python libraries on
-# the target and the ZOA Utilities Python library path separated by semi-colons,
-# for example: /usr/lpp/IBM/zoautil/lib/:/usr/lpp/rsusr/python36/lib:/lib:/usr/lib:..
+#   the target and the ZOA Utilities Python library path separated by
+#   semi-colons, for example:
+#   /usr/lpp/IBM/zoautil/lib/:/usr/lpp/rsusr/python36/lib:/lib:/usr/lib:..
 # - The value for property PATH is the ZOA utilities BIN path and Python
-# interpreter path, for example: /usr/lpp/IBM/zoautil/bin:/usr/bin/python:/bin
+#   interpreter path, for example:
+#   /usr/lpp/IBM/zoautil/bin:/usr/bin/python:/bin
+# - The value for the property **_CEE_RUNOPTS** is the invocation Language
+#   Environment® runtime options for programs and used by Python, for example,
+#   'FILETAG(AUTOCVT,AUTOTAG) POSIX(ON)'.
+# - The value for properties __TAG_REDIR_ERR, _TAG_REDIR_IN, _TAG_REDIR_OUT
+#   is txt and should be used when your shell is bash; for example:
```

Comment:
Same comment as above, don't think this is accurate since we do not support bash as the shell on managed node and variables are meaningless on control node.